guides/source/caching_with_rails.md

```diff
@@ -129,6 +129,15 @@ All cached templates from previous renders will be fetched at once with much
 greater speed. Additionally, the templates that haven't yet been cached will be
 written to cache and multi fetched on the next render.
 
+If your collection cache depends on multiple sources (try to avoid this to keep things simple),
+you can name all these dependencies as part of a block that returns an array:
+
+```html+erb
+<%= render partial: 'products/product', collection: @products, cached: -> product { [ product, current_user ] } %>
```

Comment:
Can you add parenthesis to the lambda parameter and remove the extra spacing on the array to be consistent with the rest of the page?

Reply:
I don’t think “try to avoid this to keep things simple” cuts it for a guide, then we may as well just leave this to the documentation on the CollectionCache module.

Reply:
I'm also trying to get at a larger point. Guides are meant to present an overall concept, like a kind of story, and not just every obscure feature under the sun.

I'm leaning towards no for this inclusion. @fxn usually knows more about the Guides voice, so I'd like to hear his opinion too.

Reply:
Updated!

Ok, I get your point. But could you point me to a better place to get this better documented / exposed? I really had trouble to find the documentation about it ... :cry: